go/gregor/mem_sm.go

```diff
@@ -1,4 +1,4 @@
-package storage
+package gregor
```

Comment:
Yeah, true. I Just stuck it in there initially and made it work around that initial decision. The renaming was just a `s/gregor\.//g` so I didn't think too much about it. I can change it pretty easily.